backend/bottari/src/test/java/com/bottari/service/BottariTemplateServiceTest.java

```diff
@@ -93,4 +96,72 @@ void create_Exception_DuplicateItemNameInRequest() {
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessage("중복된 물품이 존재합니다.");
     }
+
+    @DisplayName("보따리 템플릿으로 보따리를 생성한다.")
+    @Test
+    void createBottari() {
+        // given
+        final Member templateOwner = new Member("owner_ssaid", "owner_name");
+        entityManager.persist(templateOwner);
+        final BottariTemplate bottariTemplate = new BottariTemplate("title", templateOwner);
+        entityManager.persist(bottariTemplate);
+        final BottariTemplateItem bottariTemplateItem1 = new BottariTemplateItem("item1", bottariTemplate);
+        entityManager.persist(bottariTemplateItem1);
+        final BottariTemplateItem bottariTemplateItem2 = new BottariTemplateItem("item2", bottariTemplate);
+        entityManager.persist(bottariTemplateItem2);
```

Comment:
```suggestion
        final Member templateOwner = new Member("owner_ssaid", "owner_name");
        entityManager.persist(templateOwner);
        
        final BottariTemplate bottariTemplate = new BottariTemplate("title", templateOwner);
        final BottariTemplateItem bottariTemplateItem1 = new BottariTemplateItem("item1", bottariTemplate);
        final BottariTemplateItem bottariTemplateItem2 = new BottariTemplateItem("item2", bottariTemplate);
        entityManager.persist(bottariTemplate);
        entityManager.persist(bottariTemplateItem1);
        entityManager.persist(bottariTemplateItem2);
```

이런 방식은 어떤가요?
persist가 사이사이에 들어있어서 읽기 어려운 것 같아서요.